monocle/db.py

```diff
@@ -310,6 +352,28 @@ class FortSighting(Base):
     )
 
 
+class RaidSighting(Base):
+    __tablename__ = 'fort_raids'
+    id = Column(Integer, primary_key=True)
+    fort_id = Column(Integer, ForeignKey('forts.id'))
+    raid_battle_ms = Column(Integer, index=True)
+    raid_spawn_ms = Column(Integer, index=True)
+    raid_end_ms = Column(Integer, index=True)
+    raid_level = Column(Integer)
+    complete = Column(TINY_TYPE)
+    pokemon_id = Column(TINY_TYPE)
+    cp = Column(TINY_TYPE)
+    move_1 = Column(SmallInteger)
+    move_2 = Column(SmallInteger)
```

Comment:
do the moves matter at all? as everyone will get something else after catching it